src/styles/index.ts

```diff
@@ -4688,7 +4688,7 @@ const styles = (theme: ThemeColors) =>
         moneyRequestImage: {
             height: 200,
             borderRadius: 16,
-            margin: 20,
+            marginHorizontal: 20,
```

Comment:
`moneyRequestImage` is only used in MoneyRequestConfirmationListFooter, and we don't want vertical margin here.